gradle-android-command-plugin/src/main/groovy/com/novoda/gradle/command/AndroidCommandPluginExtension.groovy

```diff
@@ -5,12 +5,16 @@ import org.gradle.api.Project
 
 public class AndroidCommandPluginExtension {
 
+    static final String CATEGORIES_DEFAULT = ''
```

Comment:
Yeah from the monkey docs an empty category is treated like the default categories. Right now the default is category of launcher or monkey.